.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+# Description of PR purpose/changes
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Optional Fixes # (issue)
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
```

Comment:
I think this section could maybe be `How to Test this PR` and have it include instructions for how reviewers can test the PR. This is just a nit pick though. You definitely can leave it which ever way you prefer.

Reply:
Changed it a bit